package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@inventive/sdk",
   "description": "SDK library for types, function calls and React components to embed Inventive content in any web app",
   "author": "Inventive X",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
this is a nit since it doesn't really affect us very much when these repos are internally used, but to match how versioning is typically done, I think going forward we should increment the minor version instead of the bugfix version for this repo as well as core-types. Our changes are generally not bugfix-level changes, they're typically minor (if not even major) changes.

Reply:
I tried 0.0.0.1, but it said it was an invalid version. 🤔

Reply:
Maybe I had a typo at that time, 0.0.1.1 works. :)